src/Services/Device.vala

```diff
@@ -265,4 +265,99 @@ public class Power.Services.Device : Object {
 
         return "battery-full";
     }
+
+    public string get_info () {
+        var percent = (int)Math.round (percentage);
+        var charging = is_charging;
+
+        if (percent <= 0) {
+            return _("Calculating…");
+        }
+
+        var info = "";
+
+        if (charging) {
+            info += _("%i%% charged").printf (percent);
+
```

Comment:
If `percent` >= 100 then can skip rest of this clause

Reply:
It looks like we already do skip the rest if `time_to_full` isn't greater than 0. Do we need to check both?

Reply:
As we are rounding to the nearest integer, `percent` could equal 100 while `time_to_full` is  greater than zero, when `percent` is >= 99.5.  Its not a big issue though.